### DISCUSSION TITLE
how can use openvslam create dense map?

### DISCUSSION THREAD

**Discussion by jcyhcs** [stella-cv/stella_vslam#247] [Q&A] [answered]
hi,professor:
how can use openvslam to create dense map? like this:
![image](https://user-images.githubusercontent.com/34532224/148173204-9fa76b62-3393-402c-ae07-2961fd472d36.png)
please help, thanks!

**Comment by ymd-stella** [comment, marked as answer]
I am not interested in dense mapping at this time.
